src/com/hotmail/kalebmarc/textfighter/domain/GameEntity.java

```diff
@@ -0,0 +1,27 @@
+package com.hotmail.kalebmarc.textfighter.domain;
+
+
+public class GameEntity{
+    
+  private static double bankInterest;
+  private static int bankBalance;
+
+  public double getBankInterest(){
+    return bankInterest;
+  }
+  
+  
+  public void setBankInterest(double bankInterest){
+    GameEntity.bankInterest = bankInterest;
```

Comment:
You do not need to prefix this with `GameEntity` since you are already in that class

Reply:
I will give you new entity class